src/app/shared/host-window.service.ts

```diff
@@ -92,4 +92,10 @@ export class HostWindowService {
       distinctUntilChanged()
     );
   }
+
+  isMobileView(): Observable<boolean> {
```

Comment:
The fact that `width < MD_MIN` doesn't really say anything about it being mobile or not. All it does it check wether isXs() or isSm() is true. So I'd call it `isXsOrSm()`, and if it's only needed in `auth-nav-menu.component` I'd move it there

Reply:
something similar it's used [here](https://github.com/4Science/dspace-angular/blob/43380732def7a92845fa36a0b4b4d95e16a8caa5/src/app/%2Bsearch-page/search-page.component.ts#L55)  and [here](https://github.com/4Science/dspace-angular/blob/43380732def7a92845fa36a0b4b4d95e16a8caa5/src/app/%2Bsearch-page/search-sidebar/search-sidebar.service.ts#L18). Should we use only this method in the service after renaming it?

Reply:
That'd be great, yes